pillar.example

```diff
@@ -87,3 +87,12 @@ proftpd:
     ClamAV: '"on"'
     # Specify the UNIX Local Socket
     ClamLocalSocket: /var/run/clamav/clamd.sock
+    #Use it to manage modules by salt
+    OverrideDefaultModules: False
```

Comment:
for this setting, because it's not a ProFtpd directive, I think it should be named differently in snake case better than camel case.

And maybe put elsewhere.

Reply:
like in a `config` section or something like this. Let's discuss that with other reviewers.

Reply:
Snake case done
Lets talk about config section